main.py

```diff
@@ -99,18 +102,34 @@ async def run(
     print(f'[bold cyan]🚀 Starting benchmark run: {run_number}/{runs}...[/bold cyan]')
 
     # Go to URL
-
+    print(
+        f'🚀  Running benchmark for approach: {approach}, dataset: {dataset}, zarr_version: {zarr_version} on {url} 🚀'
+    )
     await page.goto(url)
 
-    # Focus on and click the map element
-    await asyncio.gather(page.focus('.mapboxgl-canvas'), page.click('.mapboxgl-canvas'))
+    # select approach in radio input
+    await page.click(f'label:has(input[value="{approach}"])')
+
+    # select 'Zarr version' dropdown
+    await page.select_option('div:has-text("Zarr version") select', f'{zarr_version}')
+
+    # Wait for the dropdown to be visible
+    await page.wait_for_selector('text=Dataset')
+
+    # Find the select element that is a child of the div containing the 'Dataset' text
+    dataset_dropdown = await page.query_selector(
+        'xpath=//div[text()="Dataset"]/following-sibling::div//select'
+    )
+    await dataset_dropdown.select_option(dataset)
```

Comment:
Note that I just changed the prototype behavior to not select a dataset by default. Otherwise, I think we would've still run into the premature metadata requests when testing v3 + 1MB chunks because the map would've loaded as soon as the approach was selected.